### DISCUSSION TITLE
Add "Duplicated" Option for Reporting Quotes

### DISCUSSION THREAD

**Discussion by Omaeda** [monkeytypegame/monkeytype#5604] [Ideas] [answered]
While I understand that quotes are typically reviewed before being added, there are instances—such as the one depicted in the attached image—where the wording may not be identical, yet the intended meaning remains consistent.

![imagen](https://github.com/monkeytypegame/monkeytype/assets/51844985/1e1eade2-baf8-45dc-be12-9edae3165f5e)

**Comment by Miodec** [comment]
Sure

**Reply by Miodec** [reply]
Also, I couldn't find a duplicate of the quote you sent.

**Reply by Omaeda** [reply]
> Also, I couldn't find a duplicate of the quote you sent.

![imagen](https://github.com/monkeytypegame/monkeytype/assets/51844985/c7613469-50a5-4ee6-8db6-a37c1b5d78f1)

**Reply by Miodec** [reply, marked as answer]
Thanks, also added duplicate quote as a report reason.
